build-ps/percona-xtradb-cluster.spec

```diff
@@ -1311,6 +1321,22 @@ fi
 if [ -x sbin/restorecon ] ; then
   sbin/restorecon -R var/lib/mysql
 fi
+if [ "$UPDATE_REQUIRES" = "true" ] ; then
+cat >/usr/share/mysql/pxc_cluster_view.sql << EOL
```

Comment:
likewise,  why a heredoc?  Just commit pxc_cluster_view.sql and install as a package managed file.